pkg/minikube/constants/constants.go

```diff
@@ -123,7 +123,7 @@ const (
 	ExistingContainerHostEnv = MinikubeExistingPrefix + "CONTAINER_HOST"
 
 	// TimeFormat is the format that should be used when outputting time
-	TimeFormat = time.RFC1123
+	TimeFormat = "01/02/2006 15:04:05 MST"
```

Comment:
Thank you for this PR is there a standard time.RFC that we could use ?

Reply:
how about we use   time.RFC822 
which is "02 Jan 06 15:04 MST"

Reply:
@medyagh so seconds are not that important and RFC822 certainly looks better. 
for example if we start 2 clients at same minute different seconds will look as below
Mon, 18 Apr 2022 15:52:01 PDT  -> 18 Apr 2022 15:52 PDT
Mon, 18 Apr 2022 15:52:10 PDT ->18 Apr 2022 15:52 PDT
may be seconds does not have any significant value here. i will upload new patch